Client/Application/AppDelegate+Experiments.swift

```diff
@@ -10,8 +10,33 @@ import Foundation
 
 extension AppDelegate {
     func initializeExperiments() {
-        let options = Experiments.getLocalExperimentData()
-            .map(Experiments.InitializationOptions.testing(localPayload:)) ?? .normal
+
+        let defaults = UserDefaults.standard
+        let nimbusFirstRun = "NimbusFirstRun"
+        let firstRun = defaults.object(forKey: nimbusFirstRun) != nil
+        defaults.set(false, forKey: nimbusFirstRun)
```

Comment:
Suggestion: It would be nice move defaults / keys to some sort of NimbusConstants so we can keep track of them